test/make_dist.py

```diff
@@ -159,15 +168,11 @@ def download_dist():
 
 def make_dist(download_only=False):
     # first try to download a pre-generated dist tarball; this is a lot faster
-    # but these tarballs are built for production NPM mode
-    source = None
-    if os.getenv("NODE_ENV") != "development":
-        source = download_dist()
+    source = download_dist()
```

Comment:
I think we lost the logic to check for `dist/` already existing here?

Reply:
What do you mean? It's part of the checks that `download_dist()` does by itself.